_posts/2015/2016-07-20-fullstackcon-2016-retrospective.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "ePages goes FullStack JavaScript conference"
+date: "2016-07-19 14:00:00"
+image: blog-header/javascript.png
+categories: events
+authors: ["Paolo"]
+---
+
+The Brexit is coming, the pound sterling is low, and my fellow colleague Erik and me took the opportunity to visit a conference in London ~~before we'll need a visa for that~~. It was called [_FullStack - the conference on JavaScript, Node & Internet of Things_][fullstack-website] and featured, among others, speakers from Google, Microsoft, and Facebook.
+In this post, I want to share my key takeaways from three very intense conference days.
+
+## 1. JavaScript is spreading out
+
+{% imagecaption blog/blog-tessel.png %}
+An LED sound level meter built within a few hours of JavaScript hacking.
+{% endimagecaption %}
+
+With JavaScript rapidly becoming a language to program the **Internet of Things (IoT)**, quite a few talks at the conference covered topics like remote controlling [bluetooth][bluetooth-talk] and [medical devices][pacemaker-talk], or [Virtual Reality][vr-talk]. In one of the workshops around minicomputers and microcontrollers, my colleague Erik hacked an LED sound level meter using Node.js on the [Tessel 2][tessel-website].
+
+Naturally, in the IoT context **security** gains more importance, and I attended two interesting ~~advertisements~~ talks on [automated code vulnerability scanning][jacks-talk] with [Jacks][jacks-website] and [protecting your client-side JavaScript][jscrambler-talk] with [Jscrambler][jscrambler-website]. Both products provide convenient ways for developers to secure their JavaScript applications, without requiring an expertise in the domain of web application security.
+
+## 2. React still has huge momentum
+
+Just like the other conferences I attended in the last 12 months, _FullStack_ featured the canonical talk by a company that [moved from a monolithic [add your current technology here] application to a micro-service based system with a front-end powered by an isomorphic React & Redux app running on NodeJS][react-talk].
+
+In total, there were around half a dozen talks around the [NodeJS][node]/[React][react]/[Redux][redux] stack that my team is using for building ePages' next generation frontend and website editor. One of the most interesting ones was by [Forbes Lindesay][forbes-twitter] from Facebook, who talked about [data fetching in React][react-data-fetching]. All in all there was plenty of opportunity to compare notes on that topic with other attendants from the very international audience.
```

Comment:
most interesting ones was -> most interesting was

ones is commonly (and falsely) used